content/blog/2020/06/camel34-whatsnew/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Apache Camel 3.4 What's New"
+date: 2020-06-19
+authors: [davsclaus]
+categories: ["Releases"]
+preview: Whats included in the Camel 3.4 LTS release.
+---
+
+Apache Camel 3.4 is the first LTS (Long Term Support) release of Camel 3.
+
+This release will be active supported with regular patch releases (important bug and security fixes) for 1-year.
+
+For more details about LTS vs non-LTS releases see this [blog post](https://camel.apache.org/blog/LTS-Release-Schedule/).
+
+
+### So whats in this release?
+
+This release is mostly a more robust and bug fix release. 
+
+We have also continued the work to make Camel more modular and lighter. 
+This time we removed the needed for JAXB in the swagger and openapi modules.
```

Comment:
"removed the need for JAXB..." or maybe just "removed the dependency on JAXB..."